### ISSUE TITLE
Antivirus blocks hooking

### ISSUE THREAD

**Issue by sarathy365** [nektra/Deviare2#43]
Need: Identify all the applications that are opened, and kill some if needed.

While i am trying to hook explorer.exe, BitDefender(Advanced threat defense) blocks it.
Is there a way to skip this?

**Comment by mxmauro** [comment]
No. Antivirus blocks in kernel and, if you can skip this, the AV has a bug.

I think it is better to develop an explorer extension and register it in the system.

**Comment by sarathy365** [comment]
**explorer extension**

can you please provide more insight on this?

**Comment by mxmauro** [comment]
See [this](https://docs.microsoft.com/en-us/windows/desktop/shell/shell-extensibility-bumper)
